create.go

```diff
@@ -29,9 +29,9 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 			Usage: `path to the root of the bundle directory, defaults to the current directory`,
 		},
 		cli.StringFlag{
-			Name:  "console",
+			Name:  "console-socket",
 			Value: "",
-			Usage: "specify the pty slave path for use with the container",
+			Usage: "specify an AF_UNIX socket to send the master fd through",
```

Comment:
Can this be made more descriptive.. send the master fd through sounds like the perspective is the container vs. the user / client application who is receiving / attaching through the specified socket.